assets/source/setup-guide/app/components/Billing/Billing.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+import { Spinner } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import {
+	Button,
+	CardBody,
+	Flex,
+	FlexItem,
+	FlexBlock,
+	__experimentalText as Text, // eslint-disable-line @wordpress/no-unsafe-wp-apis --- _experimentalText unlikely to change/disappear and also used by WC Core
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Clicking on "Setup Billing" button.
+ *
+ * @event wcadmin_pfw_billing_setup_button_click
+ */
+
+/**
+ * Clicking on "Go to billing settings" button.
+ *
+ * @event wcadmin_pfw_go_to_billing_button_click
+ */
+
+const Billing = ( { accountData } ) => {
+	const isBillingSetup = accountData?.is_billing_setup;
+	const statusLabe = classnames( 'pfw-billing-info', {
+		'pfw-billing-info--status-success': isBillingSetup === true,
+		'pfw-billing-info--status-error': isBillingSetup === false,
+	} );
+
+	let element = null;
+
+	if ( isBillingSetup === undefined ) {
+		element = <Spinner className="billing-info__preloader" />;
+	}
+
+	if ( isBillingSetup === true ) {
+		element = (
+			<>
+				<FlexBlock className={ statusLabe }>
+					<Text variant="body">
+						{ __(
+							'Billing Setup Correctly',
+							'pinterest-for-woocommerce'
+						) }
+					</Text>
+				</FlexBlock>
+
+				<FlexItem>
+					<Button
+						variant="link"
```

Comment:
We are using older version of WordPress components, so this one isn't right. We should use `isLink` instead.

```suggestion
						isLink
```